blosc/blosc2.c

```diff
@@ -2228,7 +2240,7 @@ static void t_blosc_do_job(void *ctxt)
   /* Resize the temporaries if needed */
   if (blocksize != thcontext->tmpblocksize) {
     my_free(thcontext->tmp);
-    thcontext->tmp = my_malloc((size_t)3 * blocksize + ebsize);
+    thcontext->tmp = my_malloc((size_t) 3 * blocksize + ebsize);
```

Comment:
Regarding casting, we follow the convention of not letting a space between the cast and the variable (or constant in this case).